values.example.yaml

```diff
@@ -53,3 +53,195 @@ concourse:
     database: 'concourse'
```

Comment:
I didn't come up with this example file, just added to it. For the same reason as above, I actually don't like having this exaple file around, because that means having to maintain multiple `values.yaml` files